pilot/cmd/pilot-discovery/main.go

```diff
@@ -114,9 +114,9 @@ func init() {
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPMaxMessageSize, "mcpMaxMsgSize", bootstrap.DefaultMCPMaxMsgSize,
 		"Max message size received by MCP's grpc client")
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPInitialWindowSize, "mcpInitialWindowSize", bootstrap.DefaultMCPInitialWindowSize,
-		"Max message size received by MCP's grpc client")
+		"Initial window size for the gRPC connection")
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPInitialConnWindowSize, "mcpInitialConnWindowSize", bootstrap.DefaultMCPInitialConnWindowSize,
-		"Max message size received by MCP's grpc client")
+		"Initial connection window size for the gRPC connection")
```

Comment:
nit: `MCP's gRPC connection` may be more clear, pilot will have grpc connections for XDS as well

Reply:
Fixed. Definitely make it more clear.